docs/COMMON_ACTIONS.md

```diff
@@ -0,0 +1,20 @@
+| -------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
```

Comment:
I know we do this everywhere, but is there a reason we use so many dashes? It looks like you only need 1. `| --- | --- | --- |` would replace this entire line. It looks like some editors only need 1 (VS Code for example), so `| - | - | - |`. Personally I would love to get rid of these hyphen snakes

Reply:
@dcbroad3 @cbardini  this is more a comment for the group than just derek since all of our tables are built this way.

Reply:
trustworthy source: https://www.pluralsight.com/guides/working-tables-github-markdown